traffic_ops/traffic_ops_golang/routing/routes.go

```diff
@@ -134,6 +134,9 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		 * 4.x API
 		 */
 
+		// Queue Updates
+		{api.Version{Major: 4, Minor: 0}, http.MethodPut, `queue_updates/{cdn}?$`, server.ProfileAndTypeQueueUpdateHandler, auth.PrivLevelOperations, Authenticated, nil, 4134390571},
```

Comment:
Why a new route instead of adding `type` and `profile` query parameters to the existing `/cdns/{id}/queue_update` handler?

Reply:
It was specified in the GH issue. I'm open to changing it.

Reply:
Oh, I see that now. I'll add my thoughts to that GH issue real quick.